alg/marching_squares/utility.h

```diff
@@ -36,14 +36,14 @@ namespace marching_squares
 
 // This is used to determine the maximum level value for polygons,
 // the one that spans all the remaining plane
-const double Inf = std::numeric_limits<double>::max();
+const double Inf = std::numeric_limits<double>::infinity();
 
 const double NaN = std::numeric_limits<double>::quiet_NaN();
 
 #define debug(format, ...) CPLDebug("MarchingSquare", format, ##__VA_ARGS__)
 
 // Perturb a value if it is too close to a level value
-inline double fudge(double level, double value)
+inline double fudge(double level, double value, double minLevel)
```

Comment:
if this API can be changed maybe better if it it was
```suggestion
inline double fudge(double value, double minLevel, double level)
```

Reply:
done